lib/server/controllers/runs.controller.js

```diff
@@ -308,6 +311,26 @@ const listReasonTypes = async (_request, response, _next) => {
     }
 };
 
+// eslint-disable-next-line valid-jsdoc
+/**
+ * Retrive distinct combination of levels of alice L3 and dipole current rounded to kilo amperes
+ */
+const listAllAliceL3AndDipoleLevelsForPhysicsRuns = async (req, res) => {
+    const validatedDTO = await dtoValidator(
+        DtoFactory.queryOnly(),
+        req,
+        res,
+    );
```

Comment:
You don't need this

Reply:
it's agains other controllers in which excessive parameters cause request failure